accounts/views.py

```diff
@@ -33,7 +36,31 @@ def activate(request, activation_key):
     profile = get_object_or_404(User, activation_key=activation_key)
     if profile.akey_expires < timezone.now():
         return render_to_response('accounts/activate.html', {'expired': True})
-    profile.is_active = True
+    profile.active = True
     profile.activation_key = ''
     profile.save()
```

Comment:
Dobry nawyk, ale przypadki w których się takie coś może wydarzyć jest rzadkie, więc to co jest tutaj jest ok.